### PR TITLE
feat: Connect stories

### PR DESCRIPTION
**Pull request by gbarkhatov**
Adds 2 more stories:
- First shows the data from the both wallets https://i.imgur.com/Wpf3pDm.png
- Second allows our BTC wallet to `signPsbt` and `signMessage` https://i.imgur.com/PevsMTq.png https://i.imgur.com/djU9Ucj.png

Useful for manual testing wallets methods and properties

Keplr will be in another PR and not a priority as of today. Created an issue for that:
https://github.com/babylonlabs-io/bbn-wallet-connect/issues/142

### CODE REVIEW

**Review by totraev — CHANGES_REQUESTED** [review]
Can be simplified

File: src/components/WalletProvider/index.stories.tsx [hunk 1]
```diff
@@ -34,3 +38,242 @@ export const Default: Story = {
...
+                  </div>
+                )}
+              </div>
+            </div>
+          )}
+        </div>
+      </div>
+    );
+  },
+};
```
Comment (totraev) [inline src/components/WalletProvider/index.stories.tsx:279]:
```suggestion
export const WithBTCSigningFeatures: Story = {
  args: {
    onError: console.log,
  },
  decorators: [
    (Story) => (
      <ScrollLocker>
        <WalletProvider context={window.parent} config={config} onError={console.log}>
          <Story />
        </WalletProvider>
      </ScrollLocker>
    ),
  ],
  render: () => {
    const { open, selectedWallets } = useWidgetState();
    const [messageToSign, setMessageToSign] = useState("");
    const [psbtToSign, setPsbtToSign] = useState("");
    const [signedMessage, setSignedMessage] = useState("");
    const [signedPsbt, setSignedPsbt] = useState("");
    const [transaction, setTransaction] = useState("");

    const btcProvider = selectedWallets.BTC?.provider as IBTCProvider | undefined;

    const handleSignMessage = async () => {
      if (!btcProvider || !messageToSign) return;

      try {
        const signature = await btcProvider.signMessage(messageToSign, "ecdsa");
        console.log("handleSignMessage:", signature);
        setSignedMessage(signature);
      } catch (error) {
        console.error("Failed to sign message:", error);
      }
    };

    const handleSignPsbt = async () => {
      if (!btcProvider || !psbtToSign) return;
      try {
        const signedPsbt = await btcProvider.signPsbt(psbtToSign);
        console.log("handleSignPsbt:", signedPsbt);
        setSignedPsbt(signedPsbt);
      } catch (error) {
        console.error("Failed to sign PSBT:", error);
      }
    };

    return (
      <div>
        <Button className="b-mb-4" onClick={open}>
          Connect Wallet
        </Button>

        <div className="b-flex b-flex-col b-gap-4">
          {btcProvider && (
            <div className="b-flex b-flex-col b-gap-4">
              <div className="b-rounded b-border b-border-secondary-main/30 b-p-4">
                <FormControl label="Sign Message" className="b-mb-2 b-py-2">
                  <Input
                    type="text"
                    value={messageToSign}
                    onChange={(e) => setMessageToSign(e.target.value)}
                    placeholder="Enter message to sign"
                  />
                </FormControl>

                <Button onClick={handleSignMessage}>Sign Message</Button>

                {signedMessage && (
                  <div className="b-mt-2 b-flex b-items-center b-gap-2">
                    <Text variant="body2" className="b-flex-1 b-truncate">
                      Signed Message: {signedMessage}
                    </Text>
                    <Button onClick={() => setSignedMessage("")}>Delete</Button>
                  </div>
                )}
              </div>

              <div className="b-rounded b-border b-border-secondary-main/30 b-p-4">
                <FormControl label="Sign PSBT" className="b-mb-2 b-py-2">
                  <Input
                    type="text"
                    value={psbtToSign}
                    onChange={(e) => setPsbtToSign(e.target.value)}
                    placeholder="Enter PSBT hex"
                  />
                </FormControl>

                <Button onClick={handleSignPsbt}>Sign PSBT</Button>

                {signedPsbt && (
                  <div className="b-mt-2 b-flex b-items-center b-gap-2">
                    <Text variant="body2" className="b-flex-1 b-truncate">
                      Signed PSBT: {signedPsbt}
                    </Text>
                    <Button
                      onClick={() => {
                        setTransaction("");
                        setSignedPsbt("");
                      }}
                    >
                      Delete
                    </Button>
                  </div>
                )}

                {signedPsbt && (
                  <div className="b-mt-2 b-flex b-items-center b-gap-2">
                    <Text variant="body2" className="b-flex-1 b-truncate">
                      Transaction: {transaction}
                    </Text>
                    <Button
                      onClick={() => {
                        if (!signedPsbt) return;

                        try {
                          const tx = Psbt.fromHex(signedPsbt).extractTransaction().toHex();
                          console.log("Extracted transaction:", tx);
                          setTransaction(tx);
                        } catch (error) {
                          console.error("Failed to extract transaction:", error);
                        }
                      }}
                    >
                      Extract transaction
                    </Button>
                  </div>
                )}
              </div>
            </div>
          )}
        </div>
      </div>
    );
  },
};
```

File: src/components/WalletProvider/index.stories.tsx [hunk 2]
```diff
@@ -1,6 +1,10 @@
-import { Button, ScrollLocker } from "@babylonlabs-io/bbn-core-ui";
+import { Button, ScrollLocker, Text } from "@babylonlabs-io/bbn-core-ui";
```
Comment (totraev) [inline src/components/WalletProvider/index.stories.tsx:1]:
```suggestion
import { Button, FormControl, Input, ScrollLocker, Text } from "@babylonlabs-io/bbn-core-ui";
```

**Review by totraev — CHANGES_REQUESTED** [review]

**Comment by totraev** [comment]
> This is storybook only right? Could i confirm this won't be included in the package build?

Correct

**Comment by gbarkhatov** [comment]
@jrwbabylonlab this is manual for storybook, correct. It allows to easily verify both wallets data and BTC wallet functionality (signPsbt, signMessage)

**Review by totraev — APPROVED** [review]

**Pull request merged**